### DISCUSSION TITLE
Graph card displaying 1h, 6h, 12h, 1d, ...

### DISCUSSION THREAD

**Discussion by Fusseldieb** [home-assistant/frontend#11201] [Other feature requests]
**Is your feature request related to a problem? Please describe.**
The default frontend graph (see below) is over a hardcoded time period of 24h, which makes it difficult to see changes in the last minutes, especially on mobile.

![image](https://user-images.githubusercontent.com/4715129/68221575-64ba8c00-ffc8-11e9-8b96-5bdc8d651230.png)

**Describe the solution you'd like**
A great solution would be to have buttons underneath the graph to select between 1h, 6h, 12h, 1d and maybe 7d.

**Comment by Heronimonimo** [comment]
+1 on implementing this feature request also in the Lovelace history-graph card. That would remove the necessity of changing around the update interval all the time.
